cmd/kubeadm/app/apis/kubeadm/v1beta3/conversion.go

```diff
@@ -34,3 +35,27 @@ func Convert_v1beta3_InitConfiguration_To_kubeadm_InitConfiguration(in *InitConf
 	err = Convert_v1beta3_ClusterConfiguration_To_kubeadm_ClusterConfiguration(&ClusterConfiguration{}, &out.ClusterConfiguration, s)
 	return err
 }
+
+// Convert_v1beta3_ControlPlaneComponent_To_kubeadm_ControlPlaneComponent is required due to the missing ControlPlaneComponent.ExtraEnvs in v1beta3.
+// TODO: remove this with v1beta3 https://github.com/kubernetes/kubeadm/issues/2890
```

Comment:
the todo is not needed, given the package folder will be removed as a whole. same in other new conversion functions.